DevOps.Util/HelixServer.cs

```diff
@@ -40,7 +45,7 @@ public static IHelixApi GetHelixApi(string? helixBaseUri = null, string? token =
             return ApiFactory.GetAuthenticated(helixBaseUri, authToken.Token);
         }
 
-        public async ValueTask GetHelixPayloads(string jobId, List<string> workItems, string downloadDir, bool ignoreDumps)
+        public async ValueTask GetHelixPayloads(string jobId, List<string> workItems, string downloadDir, bool ignoreDumps, bool extract)
```

Comment:
I noticed that this library doesn't try to maintain binary compatibility, so I've just reordered the parameters and ifdef'ed the signature to only expose `extract` option on `net7.0`.  This will still be source compatible and avoid exposing the extract option.